ivy/functional/ivy/data_type.py

```diff
@@ -221,6 +221,13 @@ def _get_dtypes(fn, complement=True):
             if isinstance(dtypes, dict):
                 dtypes = dtypes.get(ivy.current_backend_str(), base)
             ivy.utils.assertions.check_isinstance(dtypes, tuple)
+            if dtypes == ():
+                dtypes = base
+                logging.warning(
+                    "%s decorator can be outdated. Consider to raise an issue or       "
+                    "             contribute!",
```

Comment:
Can you change this warning to "All valid dtypes will be used because no supported dtypes detected, perhaps the unsupported dtypes decorators need to updated with the latest version"

Reply:
sure